app/javascript/react/components/capes/CapesList.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import CapeTile from './CapeTile'
+
+const CapesList = (props) => {
+  const [capes, setCapes] = useState([])
+
+  useEffect(() => {
+    fetch("/api/v1/capes.json")
+    .then(response => {
+      if (response.ok) {
+        return response
+      } else {
+        let errorMessage = '${response.status} (${response.statusText})',
+          error = new Error(errorMessage);
+          throw(error)
+      }
+    })
+    .then(response => response.json())
+    .then(response => {
+      setCapes(response)
+    })
+    .catch(error => console.error('Error in fetch: ${error.message}'))
+  }, [])
+
+  const capesTiles = capes.map((cape) => {
+    return (
+      <CapeTile
+        key={cape.id}
+        id={cape.id}
+        name={cape.name}
+        full_name={cape.full_name}
+        gender={cape.gender}
+        affiliation={cape.affiliation}
+        intelligence={cape.intelligence}
+        strength={cape.strength}
+        speed={cape.speed}
+        url={cape.url}
+      />
```

Comment:
this is a LOT of props to pass. Just pass one prop: cape=cape, and then unpack them in your CapeTile component

Reply:
Nick, we've been trying to pass this down for a couple hours and it's breaking all of our tests. Is it possible for us to continue passing down these props? We've tried refactoring our tests extensively with no luck, and Zain/Vinh advised they ran into the same problem and solved it by passing props down individually. Thanks in advance for your guidance!